code/game/objects/items/devices/flash.dm

```diff
@@ -257,3 +230,41 @@
 	limited_conversions = REVSQUAD_FLASH_USES
 	mech_flags = MECH_SCAN_FAIL
 */
+
+/obj/item/device/flash/rev
+	mech_flags = MECH_SCAN_FAIL
+	var/limited_conversions = -1
+
+/obj/item/device/flash/rev/attack(mob/living/M,mob/user)
```

Comment:
Does this mean only these special flashes can turn people into revs now? Doesn't this break being able to flash people with other flashes?

Reply:
That was never a feature, just plain shitcode.